hack/util.sh

```diff
@@ -443,14 +448,14 @@ reset_tmp_dir() {
 	local sudo="${USE_SUDO:+sudo}"
 
 	set +e
-	${sudo} rm -rf ${BASETMPDIR} &>/dev/null
+	${sudo} rm -rf "${BASETMPDIR}" &>/dev/null
 	if [[ $? != 0 ]]; then
 		echo "[INFO] Unmounting previously used volumes ..."
-		findmnt -lo TARGET | grep ${BASETMPDIR} | xargs -r ${sudo} umount
-		${sudo} rm -rf ${BASETMPDIR}
+		findmnt -lo TARGET | grep "${BASETMPDIR}" | xargs -r "${sudo}" umount
```

Comment:
Quoting `${sudo}` here means it will expand to an empty parameter if it is empty, which will cause `xargs` to fail:

```console
$ empty=
$ echo foo | xargs -r $empty /bin/echo
foo
$ echo foo | xargs -r "$empty" /bin/echo
xargs: : No such file or directory
$
```